n3fit/src/n3fit/layers/msr_normalization.py

```diff
@@ -96,8 +98,10 @@ def call(self, pdf_integrated, photon_integral):
         divisors = op.gather(y, self.divisor_indices, axis=0)
 
         # Fill in the rest of the flavours with 1
+        # (Note: using y.shape in the output_shape below gives an error in Python 3.11)
```

Comment:
```suggestion
```

I think it is even better like this tbh. Out of curiosity, is the error due to python 3.11 or 3.11 getting a newer version of tf?

Reply:
I'm not sure to be honest, the fix works regardless and it didn't look worse, so I didn't investigate further.